src/transformers/utils/quantization_config.py

```diff
@@ -146,3 +146,6 @@ def to_dict(self) -> Dict[str, Any]:
         """
         output = copy.deepcopy(self.__dict__)
         return output
+
+    def __repr__(self) -> str:
+        return f"{self.__class__.__name__}({', '.join([f'{k}={v}' for k, v in self.__dict__.items()])})"
```

Comment:
Let's match the [`__repr__` pattern](https://github.com/huggingface/transformers/blob/52c4276e443c305953b8dcf64e4309b54d26b06d/src/transformers/configuration_utils.py#L756) we have in our `PretrainedConfig` class. This will need a `to_json_string` method to be added too